doc/en/user/source/extensions/mapml/installation.rst

```diff
@@ -47,6 +47,33 @@ If the ``Represent multi-layer requests as multiple elements`` is checked (and t
 
 .. figure:: images/mapml_wms_multi_extent.png
 
+TiledCRS
+--------
+MapML supports 4 built-in TiledCRS:
+
+- MapML:WGS84 (or EPSG:4326)
+- MapML:OSMTILE (or EPSG:3857)
+- MapML:CBMTILE (or EPSG:3978)
+- MapML:APSTILE (or EPSG:5936)
+
+In addition, is it possible to configure custom TiledCRS based on the available WMTS GridSets.
+A new MapML TCRS Settings menu is available in the GeoServer UI on the Settings section:
+
+.. figure:: images/mapml_tcrs_menu.png
+
+
+It provides a selector containing available GridSets. Administrator can select GridSets from the left list that will be converted to TiledCRSs.
+
+
+.. figure:: images/mapml_tcrs_selector.png
+
+
+Notes:
+
+- Gridsets containing ":" character in the name won't be listed
+- Gridsets with not-numeric levels or without a common prefix won't be listed
```

Comment:
```suggestion
- Gridsets with non-numeric levels or without a common prefix won't be listed
```

Reply:
I'd also add an example of a gridset that would work (pick one of the built-in UTMs for example), e.g. UTM32WGS84Quad

Reply:
Done